src/safety_report_generator.py

```diff
@@ -0,0 +1,139 @@
+from invoke.context import Context
+from datetime import datetime
+
+import json
+import os
+
+TO_BE_DECIDED="To Be Decided"
+
+class SafetyReportGenerator:
+    """
+    The SafetyReportGenerator class deals with the functionality of generating safety reports for running containers.
+    The safety report takes the following format:
+    [
+        {
+            "package": "package",
+            "scan_status": "SUCCEEDED/FAILED/IGNORED",
+            "installed": "version",
+            "vulnerabilities": [
+                {
+                    "vid": "safety_vulnerability_id",
+                    "advisory": "description of the issue",
+                    "reason_to_ignore":"reason to ignore the vid",
+                    "spec": "version_spec"
+                },
+                ...
+            ]
+            "date":
+        }
+        ...
+    ]
+    """
+
+    def __init__(self, container_id, ignore_dict = {}):
+        self.container_id = container_id
+        self.vulns_dict = {}
```

Comment:
Please use whole words in variable names for clarity.

Reply:
Done.